shared_model/validators/field_validator.hpp

```diff
@@ -55,6 +55,12 @@ namespace shared_model {
       std::optional<ValidationError> validateAssetId(
           const interface::types::AssetIdType &asset_id) const;
 
+      std::optional<ValidationError> validateEvmHexAddress(
+          const std::string &address) const;
+
+      std::optional<ValidationError> validateBytecode(
+          interface::types::EvmCodeHexStringView input) const;
```

Comment:
Another way to view this method would be to check if the object of raw type (`string_view` here), corresponds to some additional rules. So that after calling this method, it is valid to cast the object to strong type.

Reply:
strong types also help to bind the meaning of the data, while the format (hex) may be the same.